### PR TITLE
Add Sightsea project

### PR DESCRIPTION
**Pull request by kretoslabs**
# Summary

List Sightsea project on the Moonbeam DApp Directory.

### CODE REVIEW

File: projects/sightsea/sightsea.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "id": "sightsea",
+  "slug": "sightsea",
+  "name": "sightsea",
+  "category": "Social",
+  "coinGeckoId": "chainlink",
```
Comment (themacexpert) [inline projects/sightsea/sightsea.json:6]: I don't think CoinGeckid is correct, do you want to delete that?

**Review by themacexpert — CHANGES_REQUESTED** [review]
Looks great! Just pointed one thing out with the coingecko id which you can either modify or delete.

File: projects/sightsea/sightsea.json (hunk 1 above)
Comment (kretoslabs) [reply projects/sightsea/sightsea.json]: @themacexpert Thanks for pointing out. That was a mistake from copying a template, I'm so sorry about it.

**Comment by aaron2048** [comment]
@themacexpert - can you check latest changes?

**Comment by kretoslabs** [comment]
Hi there, are they any updates on this?

**Review by themacexpert — APPROVED** [review]

**Pull request merged**